pkg/zedctr/trigger

```diff
@@ -1 +1 @@
-1528784313
```

Comment:
@srinibas-zededa in the future please make sure NOT to include this file in your commits.